src/entitytypes.js

```diff
@@ -976,6 +976,35 @@ SlugIdArray.prototype.indexOf = function(slug) {
   return -1;
 };
 
+/** Determines whether it includes a certain element, returning true or false as appropriate. */
+SlugIdArray.prototype.includes = function(slug) {
+  return this.buffer.includes(slugIdToBuffer(slug));
+};
+
+/** The shift() method removes element. */
+SlugIdArray.prototype.shift = function() {
+  if (this.length === 0) {
+    return;
+  }
+
+  this.buffer.copy(this.buffer, 0, SLUGID_SIZE);
+  this.avail  += 1;
+  this.length -= 1;
+  this.realloc();
+};
```

Comment:
Should return the element removed: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/shift